content/docs/guides/cadvisor.md

```diff
@@ -0,0 +1,123 @@
+1. The `redis` service is a standard Redis server. cAdvisor will gather container metrics from this container automatically, i.e. without any further configuration.
+
+To run the installation:
+
+```bash
+docker-compose up
+```
+
+If Docker Compose successfully starts up all three containers, you should see output like this:
+
+```
+prometheus  | level=info ts=2018-07-12T22:02:40.5195272Z caller=main.go:500 msg="Server is ready to receive web requests."
+```
+
+You can verify that all three containers are running using the [`ps`](https://docs.docker.com/compose/reference/ps/) command:
+
+```bash
+docker-compose ps
+```
+
+Your output will look something like this:
+
+```
+   Name                 Command               State           Ports
+----------------------------------------------------------------------------
+cadvisor     /usr/bin/cadvisor -logtostderr   Up      8080/tcp
+prometheus   /bin/prometheus --config.f ...   Up      0.0.0.0:9090->9090/tcp
+redis        docker-entrypoint.sh redis ...   Up      0.0.0.0:6379->6379/tcp
+```
+
+## Exploring the cAdvisor web UI
+
+You can access the cAdvisor [web UI](https://github.com/google/cadvisor/blob/master/docs/web.md) at `http://localhost:8080`. You can explore stats and graphs for specific Docker containers in our installation at `http://localhost:8080/docker/<container>`. Metrics for the Redis container, for example, can be accessed at `http://localhost:8080/docker/redis`, Prometheus at `http://localhost:8080/docker/prometheus`, and so on.
+
+## Exploring metrics in the expression browser
+
+cAdvisor's web UI is a useful interface for exploring the kinds of things that cAdvisor monitors, but it doesn't provide an interface for exploring container *metrics*. For that we'll need the Prometheus [expression browser](/docs/visualization/browser), which is available at `http://localhost:9090/graph`. You can enter Prometheus expressions into the expression bar, which looks like this:
+
+![Prometheus expression bar](/assets/prometheus-expression-bar.png)
+
+Let's start by exploring the `container_start_time_seconds` metric, which records the start time of containers (in seconds). You can select for specific containers by name using the `name="<container_name>"` expression. The container name corresponds to the `container_name` parameter in the Docker Compose configuration. The [`container_start_time_seconds{name="redis"}`](http://localhost:9090/graph?g0.range_input=1h&g0.expr=container_start_time_seconds%7Bname%3D%22redis%22%7D&g0.tab=1) expression, for example, shows the start time for the `redis` container.
+
+NOTE: A full listing of cAdvisor-gathered container metrics exposed to Prometheus can be found in the [cAdvisor documentation](https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md).
+
+## Other expressions
+
+The table below lists some other example expressions for the `redis` container:
+
+Expression | Description
+:----------|:-----------
```

Comment:
Ok, since you added a whole table now, how about adding some networking stats :)

* `rate(container_network_receive_bytes_total[1m])` for received bytes per second
* `rate(container_network_transmit_bytes_total[1m])` for transmitted bytes per second